exporters/bypasses/stream_bypass.py

```diff
@@ -4,8 +4,9 @@
 
 from exporters.bypasses.base import BaseBypass
 from exporters.module_loader import ModuleLoader
+from exporters.iterio import cohere_stream
 
-Stream = namedtuple('Stream', 'file_obj filename size')
+Stream = namedtuple('Stream', 'filename size meta')
```

Comment:
I don't get it, it seems that `meta` is empty in all `Stream` initializations.
When is meta actually set?